bitnami/elasticsearch/Chart.yaml

```
@@ -25,4 +25,4 @@ name: elasticsearch
 sources:
   - https://github.com/bitnami/bitnami-docker-elasticsearch
   - https://www.elastic.co/products/elasticsearch
-version: 15.1.6
+version: 15.1.7
```

Please bump minor version instead since you're adding new parameters:
```suggestion
version: 15.2.0
```